### ISSUE TITLE
Sequence trimming logic is confusing

### ISSUE THREAD

**Issue by ahlgorithm** [getsentry/sentry-python#409]
It's not clear in the UI that "long" sequences are being intentionally trimmed. For example, `map(str, range(20))` results in the following:

![Screen Shot 2019-07-02 at 8 51 42 PM](https://user-images.githubusercontent.com/3722355/60555503-39b6e780-9d0b-11e9-914c-43f6d4d94028.png)

It wasn't clear to me that the list was intentionally trimmed until I stumbled across #344.

Adding ellipses in these cases would provide a much better UX (as is already done with strings). The above example would instead serialize something like this:

```
[
 '0',
 '1',
 '2',
 '3',
 '4',
 '5',
 '6',
 '7',
 '8',
 '9',
 '10',
 '11',
 '12',
 '13',
 '14',
 '15',
 '16',
 ...
]
```

**Comment by untitaker** [comment]
I agree, we should fix this in the UI (the data that this array has been trimmed is already in the event payload)

**Comment by Siecje** [comment]
I see an ellipse on the Sentry hosted issue page.

**Comment by iker-barriocanal** [comment]
I believe this has been fixed in the UI, so closing this issue. Feel free to comment here if you still think we should work on this.

Thanks!

**Comment by antonpirker** [comment]
@untitaker do you know if this was addressed already in `getsentry/sentry`? Could you link the related issue here?

**Comment by untitaker** [comment]
I can't find any duplicate of this in getsentry/sentry atm, I would transfer it to that repo.
